### PR TITLE
[WD-7268] add landing page for Chiselled Ubuntu without the chart and a table

### PR DESCRIPTION
**Pull request by akbarkz**
## Done

[Copy doc](https://docs.google.com/document/d/1ZkQ-UFTwsPkMnQn8IEEjZOXMy40tkgBscw_-qq9K3nc/edit)
[Figma design](https://www.figma.com/file/YfNc8KmYqa2e7JgbFo6lXa/24.04-U.com---%2Fcontainers?type=design&node-id=96-2890&mode=design&t=B0iQTEU59HoXl9go-4)

- Add new page for Chiselled Ubuntu at `/containers/chiselled`
- Add `Chiselled Ubuntu` and `Chiselled and .NET` to navigation of `/containers`
- Existing `/containers/chiselled/dotnet` page opens from `Chiselled and .NET` navigation item
- New `/containers/chiselled` page opens from `Chiselled Ubuntu` navigation item

This PR does not contain the chart (horizontal bar chart) and the table/image from `Why use Chiselled Ubuntu containers` section, it will be delivered in a separate ticket

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Navigate to `/containers/chiselled` and test the new page
- Check the navigation at `/containers` for Chiselled items

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-7268

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

File: templates/containers/chiselled/index.html [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+    <div class="col">
+      <h2>Ultra-small, ultra-secure containerisation</h2>
+      <p>Rethink your containerisation strategy with chiselled Ubuntu—where ultra-small meets ultra-secure. Trim your
+        attack surface by 6x and get the reliability of a vendor-supported Linux distribution.</p>
+      <hr />
+      <a
+        href="https://canonical-public-cloud.readthedocs-hosted.com/en/latest/oci/oci-how-to/create-chiselled-ubuntu-image.html">Learn
+        about using chisel in production&nbsp;&rsaquo;</a>
+      <br />
+      <a href="/containers/contact-us" class="js-invoke-modal">Get support for chiselled Containers&nbsp;&rsaquo;</a>
```
Comment (jw-can) [inline templates/containers/chiselled/index.html:24]:
```suggestion
      <a href="/containers/contact-us" class="js-invoke-modal">Get support for chiselled containers&nbsp;&rsaquo;</a>
```

File: templates/containers/chiselled/index.html [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+      <img src="https://assets.ubuntu.com/v1/05f4aaaa-Frame%203363.png" alt="" />
+    </div>
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--top">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
+      <p class="p-text--small-caps u-no-margin--bottom u-sv-3">How to reduce container image size</p>
```
Comment (jw-can) [inline templates/containers/chiselled/index.html:41]:
```suggestion
      <p class="p-text--small-caps u-no-margin--bottom u-sv-3">How to reduce container image size?</p>
```

File: templates/containers/chiselled/index.html [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--top">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
+      <h2>How does Chisel work</h2>
```
Comment (jw-can) [inline templates/containers/chiselled/index.html:150]:
```suggestion
      <h2>What industry experts say</h2>
```

File: templates/containers/chiselled/index.html [hunk 4]
```diff
@@ -0,0 +1,321 @@
...
+        <div class="col-4 col-medium-3">
+          <ul class="p-list--divided">
+            <li class="p-list__item">
+              <a href="https://hub.docker.com/u/ubuntu">Ubuntu on Docker Hub</a>
+            </li>
+            <li class="p-list__item">
+              <a href="http://ubuntu/dotnet-runtime">Chiselled Ubuntu for Chiselled .NET</a>
+            </li>
+            <li class="p-list__item">
+              <a href="http://ubuntu/dotnet-deps">Chiselled Ubuntu for Chiselled .NET Deps</a>
```
Comment (jw-can) [inline templates/containers/chiselled/index.html:274]: these two links don't work

**Review by jw-can — CHANGES_REQUESTED** [review]
There are quite a few places where chiselled or chisel are written with capital C, such as:
"How does **C**hisel work", "Chiselled Ubuntu for **C**hiselled .NET"
The copy doc also had these capital C's, but judging from the discussion there it should be all lowercase c when mid-sentence

File: templates/containers/chiselled/index.html (hunk 2 above)
Comment (akbarkz) [reply templates/containers/chiselled/index.html:41]: This is according to Figma design, there is a link in the description. I will ask Sophie to apply the correction to copy doc as well.

File: templates/containers/chiselled/index.html (hunk 4 above)
Comment (akbarkz) [reply templates/containers/chiselled/index.html:274]: my bad, I just copied them from the copy doc without even checking. Requested the correct ones.

**Review by lyubomir-popov — APPROVED** [review]
small fixes in a commit, all else looks great, thanks.

**Comment by Sophie-32** [comment]
Updated the copydoc with lowercase c's for 'chisel' and/or 'chiselled'. I see the video section was updated with new text after we started working on this.

**Review by jw-can — APPROVED** [review]

**Pull request merged**